docs/modules/integrations/pages/zephyr-exporter.adoc

```diff
@@ -18,6 +18,15 @@ include::partial$jira-configuration.adoc[]
 |Required
 |Description
 
+|`zephyr.exporter.api-type`
+|false
+|The type of Jira API (`SCALE`, `SQUAD`). The `SCALE` type is selected by default.
+
+Zephyr Squad (formerly "Zephyr for Jira") provides base functionality for test case management easy to start, easy to use. https://smartbear.com/test-management/zephyr-squad/[More info]
+
+Zephyr Scale (formerly "Test Management for Jira") provides advanced test case management capabilities, such as cross-project test management solution with
+advanced test planning, reporting, and reusability features. https://smartbear.com/test-management/zephyr-scale/[More info]
```

Comment:
it's still not clear which operations are supported by exporter for each API type